cnd/src/http_api.rs

```diff
@@ -142,18 +134,12 @@ impl Serialize for Http<Chain> {
 }
 
 impl_serialize_type_with_fields!(htlc_location::Bitcoin { "txid" => txid, "vout" => vout });
-impl_serialize_http!(crate::ethereum::Address);
-impl_serialize_http!(SwapId);
-
-impl Serialize for Http<SwapProtocol> {
+impl Serialize for Http<identity::Ethereum> {
     fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
     where
         S: Serializer,
     {
-        match &self.0 {
-            // Currently we do not expose the hash_function protocol parameter via REST.
-            SwapProtocol::Rfc003(_hash_function) => serializer.serialize_str("rfc003"),
-        }
+        self.0.serialize(serializer)
```

Comment:
Forwarding doesn't really add any value, wherever we use `Http<identity::Ethereum>`, we might as well just use `identity::Ethereum` and add a stability test.

Reply:
Adding this to my todo task under 'audit serde for http_api/db'.